endpoints.json

```diff
@@ -63,5 +64,21 @@
         "article_id": 5
       }
     ]
+  },
+  "POST /api/articles/:article_id/comments": {
+    "description": "creates a new comment at a given article_id",
+    "queries": [],
+    "exampleBody": {
+      "username": "iDeletedMyOriginalAccountAndStartedThisOneWhenIRealisedThereIsNotCharacterLimitForUsernamesOnThisSiteSorryCannotHearYouOverTheSoundOfMeUsingUpAllTheCharactersSmileyFace",
+      "body": "hi"
+    },
+    "exampleResponse": {
+      "article_id": 1,
+      "author": "butter_bridge",
+      "body": "when are y'all gonna start posting recipes on this site?",
+      "comment_id": 19,
+      "created_at": "2024-11-26T12:28:45.265Z",
+      "votes": 0
+    }
```

Comment:
Really well remembered, this will really help other develops use your server... or maybe even yourself in the future 👀